GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -716,9 +729,26 @@ private static String dateCellValue(String value) {
         return ExportImportUtils.formatDateResponse(value);
     }
 
-    private static String photoCellValue(String value, String imagePrefix) {
-        return ExportImportUtils.formatImage(imagePrefix, value);
-    }
+    private static List<String> mediaCellValues(String value, boolean useQuestionId, String imagePrefix) {
+        List<String> cells = new ArrayList<>();
+        if (value.startsWith("{")) { //JSON, possibly geotagged
+            Media media = MediaResponse.parse(value);
+            String filename = media.getFilename();
+            final int filenameIndex = filename != null ? filename.lastIndexOf("/") + 1 : -1;
+            if (filenameIndex > 0 && filenameIndex < filename.length()) {
+            	cells.add(imagePrefix + filename.substring(filenameIndex));
```

Comment:
From the line 735 to here is same as whats happening in `ExportImportUtils.formatImage()`.  May I propose we eliminate the use of `ExportImportUtils.formatImage()` to make it simpler? They we would also not need two branches.  Just one if branch for the case where the geolocation is present.